arangod/Cluster/MaintenanceRestHandler.cpp

```diff
@@ -135,12 +136,14 @@ bool MaintenanceRestHandler::parsePutBody(VPackSlice const& parameters) {
     } else if (key.isString() && (key.copyString() == "properties") && value.isObject()) {
       // code here
       prop.reset(new VPackBuilder(value));
+    } else if (key.isString() && (key.copyString() == "priority") && value.isInteger()) {
```

Comment:
there is no try/catch around these vpack calls.  anything comes through bad causes a throw ... been there, seen that.  Maybe the entire function or this function's caller should do the try/catch.

Reply:
As far as I know the `is...` functions in vpack are all nothrow, and `copyString` is protected by the `isString` before and by short evaluation of &&. I will check again, but this was my thinking.

Reply:
My concert was value.getInt() ... I missed the value.isInteger() in the line above it.

Reply:
@neunhoef : correct.